s3/pkg/service.go

```diff
@@ -538,3 +538,36 @@ func (b *s3Service) GetBackendTypeByTier(ctx context.Context, in *pb.GetBackendT
 
 	return nil
 }
+func (b *s3Service) AddUploadRecord(ctx context.Context, record *pb.MultipartUploadRecord, out *pb.BaseResponse) error {
+	log.Logf("add multipart upload record")
+	err := db.DbAdapter.AddMultipartUpload(record)
+	if err != NoError {
+		return err.Error()
```

Comment:
what do u mean by `err.Error()`

Reply:
return value type of DbAdapter S3Error not errors, but we need return errors not S3Error